packages/ipfs-topology/src/ipfs-topology.ts

```diff
@@ -10,22 +10,19 @@ const BOOTSTRAP_LIST = (ceramicNetwork: Networks): Array<Multiaddr> | null => {
     case Networks.MAINNET:
       return [
         multiaddr(
-          '/dns4/go-ipfs-ceramic-private-mainnet-external.3boxlabs.com/tcp/4011/ws/p2p/QmXALVsXZwPWTUbsT8G6VVzzgTJaAWRUD7FWL5f7d5ubAL'
+          '/dns4/bootstrap-mainnet-rust-ceramic-1.3box.io/tcp/4101/p2p/12D3KooWJC1yR4KiCnocV9kuAEwtsMNh7Xmu2vzqpBvk2o3MrYd6',
```

Comment:
are we worried about losing the bootstrap connection for folks still using kubo?  Do we want to use the functionality @dav1do added to change the bootrap list based on if we are connected to kubo or c1?